uint8 motorId
int32 encoderCount